### ISSUE TITLE
Horizontal Insets of buttonInsets aren't setting for non-label cells

### ISSUE THREAD

**Issue by meteochu** [hightower/HTHorizontalSelectionList#36]
It seems like if the cell isn't a label cell, the horizontal insets (.left and .right) won't get set at all.
Referencing to the examples, if I were to do 
```Objective-C
self.textSelectionList.buttonInsets = UIEdgeInsetsMake(0.f, 50.f, 0.f, 50.f);
```
The buttons will show up like so:
![text](https://cloud.githubusercontent.com/assets/6889117/7744203/2c2cd8a6-ff56-11e4-874c-4b790dcde3f5.png)

However, if I were to do the exact same thing with non-label cells (CatsViewController for example)
```Objective-C
self.customViewSelectionList.buttonInsets = UIEdgeInsetsMake(0.f, 50.f, 0.f, 50.f);
```
...this is the returned result:
![cats](https://cloud.githubusercontent.com/assets/6889117/7744210/38f3e78c-ff56-11e4-953e-ad058ad617bf.png)

The Horizontal ButtonInsets of the image won't change. (Not 100% sure, but I think the BottomBar SelectionIndicator changes) Changing the vertical insets work, but the horizontals are forever stuck like that.

**Comment by erikackermann** [comment]
@meteochu thanks for creating this issue.  I will take a look today and see if it's a quick fix

**Comment by erikackermann** [comment]
@meteochu will you take a look at this branch here: https://github.com/hightower/HTHorizontalSelectionList/tree/bug/fix-insets-image-cells.  Is that what you would expect to happen when you set .left/.right insets?

**Comment by meteochu** [comment]
Yep. But is there any way to reduce the insets so the cells would be touching?

**Comment by erikackermann** [comment]
@meteochu I think `contentInset` makes sense for what you are trying to do.  I think the behaviour should be like this:
 - by default, the buttons are adjacent (zero padding between)
 - setting `buttonInsets` .left/.right to positive values will inset these buttons accordingly

Is that kind of what your were hoping would happen?

**Comment by erikackermann** [comment]
I have this working (what I described in my last comment):

![screen shot 2015-05-21 at 14 47 31](https://cloud.githubusercontent.com/assets/709713/7756311/5f4a89b4-ffc8-11e4-8c66-2f8c92d8fa0b.png)



Would you the buttonInsets to affect the ButtonBorder that is used to indicate selection?  This is how that looks for label buttons:

![screen shot 2015-05-21 at 14 46 15](https://cloud.githubusercontent.com/assets/709713/7756338/867dacaa-ffc8-11e4-9a43-21368e1dec7d.png)


But that might not be desirable for custom view buttons:

![screen shot 2015-05-21 at 14 46 25](https://cloud.githubusercontent.com/assets/709713/7756344/939849d6-ffc8-11e4-8f3e-8a2c4d42b69f.png)

**Comment by erikackermann** [comment]
Alright, fixed it for both (which I'm now realizing was how it worked previously, so this shouldn't break anyone's app).

Labels:
![screen shot 2015-05-21 at 14 59 25](https://cloud.githubusercontent.com/assets/709713/7756561/03f338fc-ffca-11e4-8d86-e6c55ac149a6.png)

Custom views:
![screen shot 2015-05-21 at 14 59 33](https://cloud.githubusercontent.com/assets/709713/7756564/0972a16e-ffca-11e4-8900-c5cac03f0e1b.png)

Will merge this in and cut another release shortly.

**Comment by erikackermann** [comment]
Resolved here: #37 

@meteochu let me know if you still have issues
